spec/factories/used_file.rb

```diff
@@ -0,0 +1,17 @@
+module Pageflow
+  FactoryBot.define do
+    factory :used_file, class: UsedFile do
+      transient do
+        model { nil }
+        revision { nil }
+      end
+
+      initialize_with do
+        file = build(model)
+        usage = file.usages.build(revision: revision)
+
+        UsedFile.new(file, usage)
+      end
+    end
+  end
+end
```

Comment:
How do the cases in which the `:used_file` factory is used differ from the `create_used_file` cases?